apiary_v2.apib

```diff
@@ -1807,12 +2081,143 @@ This endpoint can be used to retrieve a paginated list of the individual jobs as
 
     + Body
 
-          {}
+          {
+            "count": 99,
+            "next": "https://partner.embed.unmade.com/v2/orders/a882ac66-9374-4910-8c93-9ea45a470bd4/jobs?page=2",
+            "previous": null,
+            "results": [
+              {
+                "id": "e3dc59c3-d4ed-42bc-8d73-23dd0e5c4301",
+                "short_id": "E3DC59C3",
+                "reference": "15572690633029",
+                "product_name": "Technical Tee",
+                "product_type": "Technical Tee 313133 XL",
+                "product_code": "335644",
+                "state": "in_production",
+                "created": "2024-07-29T13:28:22.516837Z",
+                "partner_order_reference": "my-order-ref",
+                "partner_item_reference": "my-item-ref",
+                "price_per_item": "25.00",
```

Comment:
🤔  will make more sense for the price (here and on the ecommerce side) to represent them as an object with 2 keys `value` and `currency` ? Something like `"price_per_item" : { "value": 25, "currency": USD}`  It feels nicer to have them grouped

Reply:
Yeah, fine